backend/src/database/AllCapabilities/index.ts

```diff
@@ -35,6 +35,8 @@ const AllCapabilities = [
   "subcontractors_update_all",
   "subcontractors_update_mine",
   "supplier_add_one",
+  "amendments_read_all",
```

Comment:
after my next commit, we should talk about this going forward, i don't think every section within project or contracts needs to have its own role.   but this is ok for now.